CONTRIBUTING.md

```diff
@@ -1,7 +1,92 @@
-# Contributing code to Element Web
+## CLA/Sign-off for contribution
+
+### For contributing code to `element-web` repo (CLA mandatory)
 
 Everyone is welcome to contribute code to Element Web, provided that they are willing to license their contributions to Element under a [Contributor License Agreement](https://cla-assistant.io/element-hq/element-web) (CLA). This ensures that their contribution will be made available under an OSI-approved open-source license, currently licensed under Affero General Public License v3 (AGPLv3) or General Public License v3 (GPLv3) at your choice.
 
+### For contributing code to `matrix-js-sdk` repo (Sign-off mandatory)
```

Comment:
I think this bit needs to live in the js-sdk repo given its owned by another org, we shouldn't have it link to the contributing guide owned by another entity